### DISCUSSION TITLE
Admonition title has incorrect margins when admonition appears inside a table

### DISCUSSION THREAD

**Discussion by samsav** [pradyunsg/furo#677] [Potential Issue]
I stumbled upon something I believe to be a bug. In `furo 2023.5.20`, when an admonition appears inside a table, the margins on the admonition title are inconsistent with how they appear elsewhere:

![note-title-inside-table-has-incorrect-margins](https://github.com/pradyunsg/furo/assets/6501597/aa5fa0dc-4670-4648-920a-e8e46499972c)

From what I can tell, it looks like that the issue is that that the `margin` property for the admonition title gets overridden from the styling for `p` elements inside tables:

![issue-with-note-title-inside-table](https://github.com/pradyunsg/furo/assets/6501597/767be8ca-017a-4968-8dae-82b5f9e94342)

Unfortunately I don't have a live example to share -- I've only encountered the issue in my local environment when testing a newer version of Furo. Our production environment uses `furo 2020.12.9b21` and the issue doesn't happen there (see https://docs.kanzi.com/3.9.6/en/working-with/bindings/reference-for-bindings-expressions.html#multiplication for reference). The RST source for the example in the first screenshot looks like this:

```rst
.. list-table::
   :class: layout
   :widths: 1 6

   *  - **Syntax**
      - ``value1 * value2``

   *  - **Parameters**
      - .. list-table::
           :class: layout
           :widths: 1 5

           *  - ``value1``
              - color, int, float, Vector2, Vector3, Vector4, Matrix3, Matrix4, Srt2D, Srt3D, or boolean: multiplicand

           *  - ``value2``
              - color, int, float, Vector2, Vector3, Vector4, Matrix3, Matrix4, Srt2D, Srt3D, or boolean: multiplier

        .. note::

           When you use the multiplication operator, keep in mind that:

           -  Only one of the parameters can be boolean.
           -  You can multiply Srt2D only by Matrix3, and Srt3D only by Matrix4.
```

Let me know if I can provide any more information on this!

**Comment by pradyunsg** [comment]
I've resolved this locally with beebd7e676d1538891fef4ead30f79d866fe78c2.
